marshmallow/schema.py

```diff
@@ -99,6 +101,37 @@ def __new__(mcs, name, bases, attrs):
         for attr in mcs.FUNC_LISTS:
             attr_copy = copy.copy(getattr(klass, attr))
             setattr(klass, attr, attr_copy)
+
+        # Now add in the decorated processors. By doing this after constructing
+        # the class, we let standard inheritance do all the hard work.
+        mro = klass.mro()
+
+        klass.processors = defaultdict(list)
+        for attr_name in dir(klass):
```

Comment:
Do you think we need to guarantee order beyond "raw processors last when dumping, raw processors first when loading"? I feel that this would not be an especially obvious feature, and that ordering dependencies would be better expressed by putting them in the same processor, rather than depending on some guaranteed order.